assignments/hw3/app.py

```diff
@@ -0,0 +1,26 @@
+"""Main app module for HW3.
+"""
+import sys
+
+import data
+from pyspark.sql import SparkSession
+from transformers import RollingBATransform
+
+
+def main() -> int:
+    # Builda a Spark session
```

Comment:
```suggestion
    # Builds a Spark session
```

Reply:
Typo fixed in [`e172a9a`](https://github.com/timand32/bda696/pull/3/commits/e172a8a340ccb8ba32186cbd7a2b9fd5e2557ef7).